dev-qt/qtwayland/qtwayland-5.15.2-r14.ebuild

```diff
@@ -7,6 +7,7 @@ KDE_ORG_COMMIT=efe6edcaf8eba601dff99ec6ad4457c8a4442f86
 inherit qt5-build
 
 DESCRIPTION="Wayland platform plugin for Qt"
+SLOT=5/${QT5_PV} # bug 815646
```

Comment:
I'm not opposed to adding this, but the reporter said qtwayland was upgraded from 5.15.2-r1 to 5.15.2-r13, so this subslot definition won't help.

Reply:
It should as a _one off_ (b/c changes from 5/0 -> 5/${QT5_PV}), but yeah, it won't help for future breaks in patches which get imported (if that happens). Bleh.

Reply:
Yes this would be a one-time fix, for now.